src/plugins/intel_gna/src/pre_post_process/transposition_info.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "transposition_info.hpp"
+
+#include <vector>
+
+#include "log/debug.hpp"
+#include "openvino/core/model.hpp"
+#include "openvino/core/shape.hpp"
+#include "openvino/opsets/opset9.hpp"
+
+namespace ov {
+namespace intel_gna {
+namespace pre_post_process {
+
+using namespace ov::opset9;
+
+/*
+ * Convert transposition info to preprocessing model using Transpose layer
+ */
+std::shared_ptr<ov::Model> ToProcessModel(const TranspositionInfo& t_info) {
```

Comment:
I think it's enough to have a thorough description in the header.
I don't see a need to have both preprocessing.cpp/hpp and transposition_info.cpp/hpp.

Reply:
it makes sense, but the transposition_info is legacy part and I would like to keep it separately

Reply:
moved description to the header